src/PAModel/ControlTemplates/GlobalTemplates.cs

```diff
@@ -79,8 +79,8 @@ private static ControlTemplate CreateScreenTemplate(AppType type)
             template.InputDefaults.Add("Fill", "RGBA(255, 255, 255, 1)");
             template.InputDefaults.Add("Height", "Max(App.Height, App.MinScreenHeight)");
             template.InputDefaults.Add("Width", "Max(App.Width, App.MinScreenWidth)");
-            template.InputDefaults.Add("Size", "1 + CountRows(App.SizeBreakpoints) - CountIf(App.SizeBreakpoints, Value >= Self.Width)");
-            template.InputDefaults.Add("Orientation", "If(Self.Width < Self.Height, Layout.Vertical, Layout.Horizontal)");
+            template.InputDefaults.Add("Size", "1 + CountRows(App.SizeBreakpoints) - Countif (App.SizeBreakpoints, Value >= Self.Width)");
+            template.InputDefaults.Add("Orientation", "if(Self.Width < Self.Height, Layout.Vertical, Layout.Horizontal)");
```

Comment:
I'm not sure if this will continue to work... Are PowerFX expressions case-insensitive?

Reply:
Good point, thanks for catching, restoring these. Changed with other files automatically.